page/about-jquery/how-to-setup-jquery.md

```diff
@@ -0,0 +1,88 @@
+<script>{
+	"title": "How To Setup jQuery",
+	"level": "beginner"
+}</script>
+
+### jQuery: Setup
+
+This is a basic tutorial, designed to help you get started using jQuery. Start by creating the following HTML page named `index.html` and open it in the browser:
+
+```
+<!doctype html>
+<html>
```

Comment:
Altought this is not the main focus of this page, I'm for adding `lang="en"` to improve accessibility.